### ISSUE TITLE
about the shape of competition dataset(cls_extra_test_data.h)

### ISSUE THREAD

**Issue by Roywangj** [ldkong1205/PointCloud-C#5]
hi @ldkong1205 , i find the shape of the data in cls_extra_test_data.h is (24680, 724, 3).  In dim=1, the number of points in point cloud(724), it's not a normal number(1024 or more) of point cloud, and 724 is not the same as the number of points in modelnet-c dataset (2468, 1024, 3).  This may cause some confusion in training process(use 1024 points for training or 724 points).

**Comment by Roywangj** [comment]
Also, i want to know if you have different results on modelnet-c and  cls_extra_test_data.h, is the result of cls_extra_test_data.h is much lower than that of modelnet-c.  I have some confusing results the dataset of modelnet-c and cls_extra_test_data.h. When i testing my model on modelnet-c, i have 0.776 mOA, when i testing the former model on cls_extra_test_data.h, i have results of 0.651
Here is the results:
![微信图片_20220803153102](https://user-images.githubusercontent.com/48822865/182551320-63f8f707-d259-4828-aa6d-5bcec1936a8f.jpg)
![微信图片_20220803153420](https://user-images.githubusercontent.com/48822865/182551385-cb45dcb1-7dcb-403c-b406-a85490a13d9b.png)

**Comment by ldkong1205** [comment]
@Roywangj, thank you for your interest in the PointCloud-C Challenge!

For your questions:

> *"The number of points in the point cloud is 724, it's not a normal number (1024 or more) for a point cloud"*

A: This is because we use atomic corruptions like `Drop-Local` and `Drop-Global` to build the robustness evaluation sets. The number of points in the original point clouds could be changed. Please refer to more details of this on our [project page](https://pointcloud-c.github.io/details.html).

> *"This may cause some confusion in the training process (use 1024 points for training)"*

A: This should not be a problem for most commonly used models. Recall that there is usually a max pooling operation inside the classifier module, hence the varying point numbers should be unified to a fixed number before the final output.

> *"Also, I want to know if you have different results on ModelNet-C and cls_extra_test_data.h, the result of cls_extra_test_data.h is much lower than that of ModelNet-C"*

A: Yes, the results could be very different. For the PointCloud-C Challenge on CodaLab, the test data is much more challenging as they are generated by a combination of different corruptions, rather than the separated ones in ModelNet-C. We did this to differentiate the competition and the official benchmark. 

Hope the above answer your questions. Please let us know if there is anything else.

**Comment by ldkong1205** [comment]
By the way, please also note from this [issue](https://github.com/ldkong1205/PointCloud-C/issues/3) for some details in participating the PointCloud-C Challenge.

**Comment by Roywangj** [comment]
hi @ldkong1205 .
1. I have tried testing the performance of RPC using the pretrained model on dataset cls_extra_test_data.h, i got very low results. here are the results on modelnet-c and cls_extra_test_data.h, separately.Here are the results.
![微信图片_20220804094305](https://user-images.githubusercontent.com/48822865/182744947-150a0470-2327-479c-a036-4aee308a1ee0.jpg)

![微信图片_20220804094649](https://user-images.githubusercontent.com/48822865/182745378-8f6abca7-2c99-49d3-b4a5-25f3d740129a.jpg)


2. Would you please offer the base codes of testing model on cls_extra_test_data.h (that can reproduce the results of modelnet-c on PointCloud-C Challenge)? 
![微信截图_20220804094757](https://user-images.githubusercontent.com/48822865/182745577-155fd78e-822c-4976-86a5-144731701d20.png)

**Comment by jiawei-ren** [comment]
> 1. I have tried testing the performance of RPC using the pretrained model on dataset cls_extra_test_data.h, i got very low results. here are the results on modelnet-c and cls_extra_test_data.h, separately.Here are the results.

I have evaluated the pretrained RPC on the challenge test set. The result should be 0.633995

> 2. Would you please offer the base codes of testing model on cls_extra_test_data.h (that can reproduce the results of modelnet-c on PointCloud-C Challenge)?

Please see the example evaluation code for RPC [here](https://github.com/ldkong1205/PointCloud-C/blob/main/evaluate_cls_extra_test.py). The 0.797 OA on the leaderboard is obtained with the pretrained GDANet+WolfMix.

**Comment by jiawei-ren** [comment]
CodaLab page has been updated.

**Comment by Roywangj** [comment]
hi @jiawei-ren @ldkong1205 , thanks a lot for your patitent reply. Have a nice day!

**Comment by Roywangj** [comment]
hi @jiawei-ren , do you have the plan to release the codes of producing mixed corrupted data, just like the data in competition. I have only find the codes of creating corrputed data of all levels, while codes of producing mixex data not found. I want to produce some mixed data of my own for testing the further robetness of my model. Thanks

**Comment by jiawei-ren** [comment]
The code to produce the testing data for the challenge will remain private in order to keep the corruptions out of distribution.

**Comment by Roywangj** [comment]
Okay. Thanks for your reply.
